test/unit/math/prim/mat/meta/VectorViewMvt_test.cpp

```diff
@@ -0,0 +1,51 @@
+#include <stan/math/prim/mat.hpp>
+#include <gtest/gtest.h>
+#include <iostream>
+#include <stdexcept>
+#include <vector>
+
+// Just test the use case where a single matrix is passed in to VectorViewMvt.
+// The case where a std::vector of matrices is passed in is broken and will be refactored.
```

Comment:
Ah, my bad - when I removed the other tests that hit on broken functionality I forgot there wasn't anything else covering the `Matrix` case. Will add those in as I believe `VectorViewMvt` is intended to support those.